src/Cache/InvalidCacheKeyException.php

```diff
@@ -0,0 +1,7 @@
+<?php
+namespace ScriptFUSION\Porter\Cache;
+
+class InvalidCacheKeyException extends \DomainException implements \Psr\Cache\InvalidArgumentException
```

Comment:
What is the meaning of `DomainException`? Normally I would expect `RuntimeException`.